tools/toolchains/bash/build.sh

```diff
@@ -26,7 +26,7 @@ git clean -dxf
 git reset HEAD --hard
 
 echo "Building bash with gcc ..."
-./configure
+CC="gcc -fPIE -pie" CXX="g++ -fPIE -pie" ./configure
```

Comment:
Can you please confirm that "-pie" correct? I remember it is "-fpie".

Reply:
"-fPIE" is for GCC compile, "-pie" is for link, usually they are used together.
And "-fPIE" is almost the same with "-fpie".
Details please see [`link`](https://gcc.gnu.org/onlinedocs/gcc/Code-Gen-Options.html)